player/lua/select.lua

```diff
@@ -345,6 +345,52 @@ mp.add_key_binding(nil, "select-audio-device", function ()
     })
 end)
 
+mp.add_key_binding(nil, "select-history", function ()
+    local watch_later_dir = mp.get_property("watch-later-dir")
+    watch_later_dir = mp.command_native({"expand-path",
+        watch_later_dir ~= "" and watch_later_dir or "~~state/watch_later"})
```

Comment:
This should be exported as property that uses `mp_get_playback_resume_dir` for real directory, instead re implementing the logic here.